frontend/packages/dev-console/src/components/topology/data-transforms/data-transformer.ts

```diff
@@ -125,7 +125,12 @@ export const transformTopologyData = (
   const allResourceTypes = [...allowedResources, ...kubevirtAllowedResources];
   const allResourcesList = _.flatten(
     allResourceTypes.map((resourceKind) => {
-      return resources[resourceKind] ? resources[resourceKind].data : [];
+      return resources[resourceKind]
+        ? resources[resourceKind].data.map((res) => ({
+            kind: resources[resourceKind].kind,
```

Comment:
While working on a future issue, I found these items should have both `kind` and `apiVersion` (if it exists). The kind in the resources also may not be correct `kind` if the kind is actually a `GroupVersionKind`. It can be determined via:
```
            let { kind } = resources[resourceKind];
            let apiVersion;
            if (isGroupVersionKind(resources[resourceKind].kind)) {
              kind = kindForReference(resources[resourceKind].kind);
              apiVersion = apiVersionForReference(resources[resourceKind].kind);
            }
```